components/organisms/Contributors/contributors.tsx

```diff
@@ -183,31 +183,30 @@ const Contributors = ({ repositories }: ContributorProps): JSX.Element => {
         updateLimit={setLimit}
         metaInfo={meta}
         entity="Contributors"
-        range={range}
-        setRangeFilter={store.updateRange}
         title="Contributors"
         layout={layout}
         onLayoutToggle={() => setLayout((prev) => (prev === "list" ? "grid" : "list"))}
       />
 
       {layout === "grid" ? (
-        <div className="grid w-full gap-3 grid-cols-automobile md:grid-cols-autodesktop">
-          {isLoading ? <SkeletonWrapper height={210} radius={12} count={9} /> : ""}
-          {isError ? <>An error occurred!..</> : ""}
-          {contributorArray.map((contributor, index) => (
-            <ContributorCard
-              key={index}
-              className=""
-              contributor={{ ...contributor }}
-              topic={topic}
-              repositories={repositories}
-              range={range}
-            />
-          ))}
-        </div>
+        <ClientOnly>
+          <div className="grid w-full gap-3 grid-cols-automobile md:grid-cols-autodesktop">
+            {isLoading ? <SkeletonWrapper height={210} radius={12} count={9} /> : ""}
+            {isError ? <>An error occurred!..</> : ""}
+            {contributorArray.map((contributor, index) => (
+              <ContributorCard
+                key={index}
+                className=""
+                contributor={{ ...contributor }}
+                topic={topic}
+                repositories={repositories}
+              />
+            ))}
+          </div>
+        </ClientOnly>
```

Comment:
refactor (blocking): render as much markup server side that doesn't contribute to a hydration error.

```suggestion
        <div className="grid w-full gap-3 grid-cols-automobile md:grid-cols-autodesktop">
          <ClientOnly>
            {isLoading ? <SkeletonWrapper height={210} radius={12} count={9} /> : ""}
            {isError ? <>An error occurred!..</> : ""}
            {contributorArray.map((contributor, index) => (
              <ContributorCard
                key={index}
                className=""
                contributor={{ ...contributor }}
                topic={topic}
                repositories={repositories}
              />
            ))}
          </ClientOnly>
        </div>
```